src/runtime_src/core/tools/xbutil2/OO_HostMem.cpp

```diff
@@ -116,8 +116,9 @@ OO_HostMem::execute(const SubCmdOptions& _options) const
     }
     enable =  boost::iequals(m_action, "ENABLE");
 
-    // Exit if ENABLE action is specified but size is not
-    if(enable && size == 0)
+    // Exit if ENABLE action is specified but size is not 
+    // or is a power of 2
+    if(enable && (size == 0 || (size & (size - 1))))
```

Comment:
I like the simple solution you did with regards to determine if a value is a power of two.   

To carry it further and have it available for others to use, please do create a new helper function, something like:
`bool isPowerOf2(const uint64_t x);`

and place it in the XBUtilities.cpp (and of course use it here :^) ).

Reply:
Good idea. Just finished the change in the latest commit.